consensus/leader_election/sortition/sortition_test.go

```diff
@@ -0,0 +1,53 @@
+package sortition
+
+import (
+	"crypto/rand"
+	"testing"
+
+	"github.com/pokt-network/pocket/consensus/leader_election/vrf"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSortitionBasic(t *testing.T) {
```

Comment:
Can you please leave a comment on how this test works? I was a bit confused by the test design since I was expecting a more basic scenario where you have a fixed number of validators and checked against the expected result.

Reply:
I refactored the tests and added comments in both sortition.go and sortition_test.go. 

Let me know if the new code is clearer to the reader.